### PR TITLE
#2559. Add tests for operators

### PR DESCRIPTION
**Pull request by sgrekhov**
Forgot about operators. Now added

### CODE REVIEW

File: LanguageFeatures/Augmentation-libraries/augmenting_types_A10_t01.dart [hunk 1]
```diff
@@ -32,14 +32,17 @@ main() {
...
   Expect.equals("M", MA().getter);
   MA().setter = "set M";
   Expect.equals("set M", _log);
+  Expect.equals(2, MA() + 2);
 
   Expect.equals("E", E.e1.method());
   Expect.equals("get E", E.e1.getter);
   E.e1.setter = "set E";
   Expect.equals("set E", _log);
+  Expect.equals(3, E.e1 + 1);
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/augmenting_types_A10_t01.dart:47]: Probably:
```suggestion
  Expect.equals(3, E.e1 + 3);
```

**Review by eernstg — COMMENTED** [review]
Looks good, just noticed one typo.

**Comment by sgrekhov** [comment]
Of course. Thank you! Fixed

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**